### DISCUSSION TITLE
pprint and right aligned column values output

### DISCUSSION THREAD

**Discussion by nikbucher** [johnkerl/miller#1313] [Q&A] [answered]
Hi

When I've a file 'data.csv' with this content:
```
a,b long name,c
1,43.71,2
119,92,778
```

Then I could do
`mlr --c2p --barred cat data.csv`
and I get this output
```
+-----+-------------+-----+
| a   | b long name | c   |
+-----+-------------+-----+
| 1   | 43.71       | 2   |
| 119 | 92          | 778 |
+-----+-------------+-----+
```

If I'd like to have the number values right aligned I could do something like this
`mlr --c2p --barred cat then format-values -i %3d -f %3.0f data.csv`
then I get
```
+-----+-------------+-----+
| a   | b long name | c   |
+-----+-------------+-----+
|   1 |  44         |   2 |
| 119 |  92         | 778 |
+-----+-------------+-----+
```

Is there a possibility to get an output like this?
```
+-----+-------------+-----+
| a   | b long name | c   |
+-----+-------------+-----+
|   1 |          44 |   2 |
| 119 |          92 | 778 |
+-----+-------------+-----+
```

Or is there maybe a simpler solution?

Thank you in advance - Niklaus

**Comment by nikbucher** [comment]
And 
`mlr --c2p --barred --ofmt %3.0f cat then format-values data.csv`
and also
`mlr --c2p --barred --ofmt %3d cat then format-values data.csv`

gives me this output:

```
+-----+-------------+-----+
| a   | b long name | c   |
+-----+-------------+-----+
| 1   |  44         | 2   |
| 119 | 92          | 778 |
+-----+-------------+-----+
```
is that correct?

**Comment by nikbucher** [comment]
I just found the `--right` flag, but that works on header and values and there is - as far as I see - no option to align right numbers only.

**Comment by johnkerl** [comment]
@nikbucher indeed -- see also #380

**Reply by nikbucher** [reply, marked as answer]
Thanks for your answer. So I vote for #380 too
